tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/VotingParticipation.java

```diff
@@ -194,11 +194,14 @@ public void put(int count, String locale, String org) {
 
     public static class OrgParticipation {
         public String[] localesWithVetters;
+        final public String[] coverageLocales;
         public String defaultCoverage;
 
         public OrgParticipation(Organization o) {
             this.org = o;
-            this.defaultCoverage = StandardCodes.make().getDefaultLocaleCoverageLevel(o).toString();
+            StandardCodes codes = StandardCodes.make();
+            this.defaultCoverage = codes.getDefaultLocaleCoverageLevel(o).toString();
+            coverageLocales = codes.getLocaleCoverageLocales(o).toArray(new String[0]);
```

Comment:
Minor, but if this doesn't change over time, best to create an immutable set.

Reply:
@macchiati  ordinarily yes. in this case it's actually a string array for the purposes of shipping this class over as JSON.  I think `StandardCodes…getLocaleCoverageLocales()` actually does create an immutable set here.